pkg/telemetry/telemetry.go

```diff
@@ -29,6 +44,33 @@ func NewTelemetry(cfg *Config) (*Telemetry, error) {
 		return nil, err
 	}
 
+	spanExporter, err := autoexport.NewSpanExporter(context.Background())
+	if err != nil {
+		return nil, err
+	}
+
+	tp := sdktrace.NewTracerProvider(
+		sdktrace.WithResource(Resource),
+		sdktrace.WithSampler(sdktrace.AlwaysSample()),
+		sdktrace.WithBatcher(spanExporter),
+	)
+	otel.SetTracerProvider(tp)
+	otel.SetTextMapPropagator(propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}, propagation.Baggage{}))
```

Comment:
Does this include the B3 propagation protocol?

Reply:
nope, just standard  https://www.w3.org/TR/trace-context/  is B3 a hard requirement ?

Reply:
Yeah, we want to support both OTEL context propagation and B3. Please install the B3 propagator, too.